app/routes/events/components/JoinEventForm.tsx

```diff
@@ -402,7 +415,7 @@ const JoinEventForm = ({
 
                         {event.activationTime && registrationOpensIn && (
                           <Button disabled={disabledButton}>
-                            {`Åpner om ${moment(registrationOpensIn.asMilliseconds()).format('mm:ss')}`}
+                            {`Åpner om ${moment(registrationOpensIn.add(1, 'second').asMilliseconds()).format('mm:ss')}`}
```

Comment:
It's ugly but it works correctly now. We add one second to the displayed time left until it opens, so that the countdown will end after showing 00:01 instead of showing 00:00 for one second. Normally it rounds down the number of seconds left.

Reply:
I get that, but I was wondering if it was more "right" to do
```suggestion
                            {`Åpner om ${moment(registrationOpensIn).add(1, 'second').asMilliseconds().format('mm:ss')}`}
```